### PR TITLE
Add : Illustration resource

### PR DESCRIPTION
**Pull request by ujjawaltyagii**
## Fixes Issue

Closes : #937 

## Changes proposed

- Added an resource which contains 900+ High quality illustrations and animations in 27 different styles. 
(Frontend > Illustrations)
## Screenshots

image

### CODE REVIEW

File: database/frontend/illustrations.json [hunk 1]
```diff
@@ -110,5 +110,12 @@
     "url": "https://github.com/Anmol-Baranwal/Awesome-Illustrations-4Projects",
     "category": "frontend",
     "subcategory": "illustrations"
+  },
+  {
+    "name": "Pixeltrue",
+    "description": "Get access to high quality illustrations and animations in different styles which are free to use for personal and commercial use.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:116]: [Proofreading]
```suggestion
    "description": "Get access to different, high-quality illustrations and animations that are free to use for personal and commercial use.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ujjawaltyagii! :) I pointed out a proofreading error in my inline comment. Other than that, you're off to a great start! :)

**Review by CBID2 — APPROVED** [review]
LGTM!

**Comment by ujjawaltyagii** [comment]
@rupali-codes

**Pull request merged**